d2compiler/compile.go

```diff
@@ -73,7 +73,7 @@ func (c *compiler) compileBoard(g *d2graph.Graph, ir *d2ir.Map) *d2graph.Graph {
 	c.compileBoardsField(g, ir, "steps")
 	if d2ir.ParentMap(ir).CopyBase(nil).Equal(ir.CopyBase(nil)) {
 		if len(g.Layers) > 0 || len(g.Scenarios) > 0 || len(g.Steps) > 0 {
-			g.BoardContainer = true
+			g.IsContainerOnly = true
```

Comment:
I actually don't like the use of Container here as it has meaning for diagrams. how about just `IsFolder`?

Reply:
Will go with `IsFolderOnly`.